[intelrdfpmathlib] Build failue

Package: intelrdfpmathlib[core]:x64-osx -> 20U2#5

**Host Environment**

- Host: x64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 1e68748a7 2023-04-17 (41 minutes ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`

**Failure logs**

```
-- Using cached IntelRDFPMathLib20U2.tar.gz.
-- Cleaning sources at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/IntelRDFPMathLib20U2.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean
-- Found external ninja('1.11.1').
-- Configuring x64-osx
-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/x64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/install-x64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/intelrdfpmathlib/portfile.cmake:24 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/install-x64-osx-dbg-out.log</summary>

```
[1/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_four_over_pi.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_four_over_pi.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_four_over_pi.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_four_over_pi.c
[2/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_bid.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_bid.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_bid.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_bid.c
[3/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_exception.c
FAILED: CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_exception.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_exception.c
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_exception.c:186:13: error: call to undeclared function 'raise'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
            DPML_SIGNAL(p);
            ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_exception.c:135:28: note: expanded from macro 'DPML_SIGNAL'
#   define DPML_SIGNAL(p)        raise(SIGFPE)
                                 ^
1 error generated.
[4/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_cbrt.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_cbrt.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_cbrt.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_cbrt.c
[5/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_int.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_int.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_int.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_int.c
[6/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_exp.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_exp.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_exp.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_exp.c
[7/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_lgamma.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_lgamma.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_lgamma.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_lgamma.c
[8/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_inv_hyper.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_inv_hyper.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_inv_hyper.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_inv_hyper.c
[9/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_inv_trig.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_inv_trig.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_inv_trig.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_inv_trig.c
[10/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_erf.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_erf.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_erf.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_erf.c
[11/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_log.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_log.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_log.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_log.c
[12/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_mod.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_mod.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_mod.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_mod.c
[13/239] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDECIMAL_CALL_BY_REFERENCE=0 -DDECIMAL_GLOBAL_EXCEPTION_FLAGS=0 -DDECIMAL_GLOBAL_ROUNDING=0 -DLINUX=1 -Dmach=1 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/. -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128 -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -w -Defi2=1 -DEFI2=1 -MD -MT CMakeFiles/intel_decimal128.dir/float128/dpml_ux_bessel.c.o -MF CMakeFiles/intel_decimal128.dir/float128/dpml_ux_bessel.c.o.d -o CMakeFiles/intel_decimal128.dir/float128/dpml_ux_bessel.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/intelrdfpmathlib/src/athLib20U2-d2a8954428.clean/LIBRARY/float128/dpml_ux_bessel.c
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "intelrdfpmathlib",
      "default-features": false,
      "features": []
    }
  ]
}

```
</details>

--- a/ports/intelrdfpmathlib/missing-includes.patch
+++ b/ports/intelrdfpmathlib/missing-includes.patch
@@ -1,0 +1,24 @@
+diff --git a/LIBRARY/src/bid_internal.h b/LIBRARY/src/bid_internal.h
+index cd08ea7..ea5d0a0 100755
+--- a/LIBRARY/src/bid_internal.h
++++ b/LIBRARY/src/bid_internal.h
+@@ -37,6 +37,7 @@
+ 
+ #include "bid_conf.h"
+ #include "bid_functions.h"
++#include <stdlib.h>
+ 
+ #define __BID_INLINE__ static __inline
+ 
+diff --git a/LIBRARY/float128/dpml_exception.c b/LIBRARY/float128/dpml_exception.c
+index d061a4c..c647732 100755
+--- a/LIBRARY/float128/dpml_exception.c
++++ b/LIBRARY/float128/dpml_exception.c
+@@ -132,6 +132,7 @@
+     !defined(wnt)
+ 
+ #   include <sys/signal.h>
++#include <signal.h>
+ #   define DPML_SIGNAL(p)	 raise(SIGFPE)
+ 
+ #else

--- a/ports/intelrdfpmathlib/portfile.cmake
+++ b/ports/intelrdfpmathlib/portfile.cmake
@@ -9,6 +9,7 @@ vcpkg_download_distfile(ARCHIVE
 vcpkg_extract_source_archive(
     SOURCE_PATH
     ARCHIVE ${ARCHIVE}
+    PATCHES missing-includes.patch
 )
 
 set(LIB_SOURCE_PATH "${SOURCE_PATH}/LIBRARY")

--- a/ports/intelrdfpmathlib/vcpkg.json
+++ b/ports/intelrdfpmathlib/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "intelrdfpmathlib",
   "version-string": "20U2",
-  "port-version": 5,
+  "port-version": 6,
   "description": "Intel(R) Decimal Floating-Point Math Library",
   "supports": "!(arm & windows)",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3326,7 +3326,7 @@
     },
     "intelrdfpmathlib": {
       "baseline": "20U2",
-      "port-version": 5
+      "port-version": 6
     },
     "io2d": {
       "baseline": "2020-09-14",

--- a/versions/i-/intelrdfpmathlib.json
+++ b/versions/i-/intelrdfpmathlib.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "7cf5496b9f4fbf09ed05be9b87e063a0094ac331",
+      "version-string": "20U2",
+      "port-version": 6
+    },
+    {
       "git-tree": "c829bd814e973cf55e135f1d9e50a36b9d73d551",
       "version-string": "20U2",
       "port-version": 5

